src/vos/vos_pool.c

```diff
@@ -988,6 +988,17 @@ vos_pool_ctl(daos_handle_t poh, enum vos_pool_opc opc, void *param)
 			pool->vp_policy_desc.params[i] = p->params[i];
 
 		break;
+	case VOS_PO_CTL_SET_SPACE_RB:
+		if (param == NULL)
+			return -DER_INVAL;
+
+		i = *((unsigned int *)param);
+		if (i >= 100 || i < 0) {
+			D_ERROR("Invalid space reserve ratio for rebuild. %d\n", i);
+			return -DER_INVAL;
+		}
+		pool->vp_space_rb = i;
```

Comment:
Just curious, what is the default % for vp_space_rb?

Reply:
default value is 0, which means no reserve for rebuild.

Reply:
(not sure what if anything to change, so just discussion)

specifying 100 or a very high percentage would probably be crazy and make the pool inoperable for IO, right? 
I guess vos_space_hold() could have a large volume of D_ERROR for things like "Insufficient rebuild reserved SCM space for Pool" (or the similar NVMe-related ERR message) - for example if the pool property value is changed while there is a lot of IO happening, and the percentage is too high?

Reply:
right, if the value is set to 100 or a very high percentage, there could lots of I/O failures with the error messages you mentioned.